src/textcode/markup.py

```diff
@@ -62,7 +63,7 @@ def is_markup(location):
     if location.endswith(extensions):
         return True
 
-    with open(location, 'rb') as f:
+    with io.open(location, 'r', encoding="ISO-8859-1") as f:
```

Comment:
Why using this `ISO-8859-1` encoding?
We are reading arbitrary files: so we typically open source as binary and we can eventually convert them to text/unicode based on their encoding. We cannot use any arbitrary encoding by default at first

Reply:
To remove ascii decode error

Reply:
that's not the right way. This has to be solved differently, possibly with chardet, unicode dammit, etc, strings, any routine we already have, but not this way.